monai/transforms/spatial/dictionary.py

```diff
@@ -337,6 +386,27 @@ def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Mapping[Hashable, np.
         for key in self.key_iterator(d):
             if self._do_transform:
                 d[key] = rotator(d[key])
+            self.push_transform(d, key, extra_info={"rand_k": self._rand_k})
+        return d
+
+    def inverse(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
+        d = deepcopy(dict(data))
+        for key in self.key_iterator(d):
+            transform = self.get_most_recent_transform(d, key)
+            # Check if random transform was actually performed (based on `prob`)
+            if transform[InverseKeys.DO_TRANSFORM.value]:
+                # Create inverse transform
+                num_times_rotated = transform[InverseKeys.EXTRA_INFO.value]["rand_k"]
+                num_times_to_rotate = 4 - num_times_rotated
```

Comment:
I guess this line 400 would be merged into the array transforms https://github.com/Project-MONAI/MONAI/issues/1749

Reply:
Are you suggesting I do it now? Or in the future?

Reply:
sorry i'm just making a reference here to the ticket for future work...